packages/expo-modules-core/android/src/test/java/expo/modules/kotlin/modules/ModuleDefinitionBuilderTest.kt

```diff
@@ -8,15 +8,28 @@ import org.junit.Assert
 import org.junit.Test
 
 class ModuleDefinitionBuilderTest {
+  private inline fun unbindModuleDefinition(block: ModuleDefinitionBuilder.() -> Unit): ModuleDefinitionData {
```

Comment:
I guess `unbind` was meant to be an adjective, so use `unbound` instead 😉 
```suggestion
  private inline fun unboundModuleDefinition(block: ModuleDefinitionBuilder.() -> Unit): ModuleDefinitionData {
```